dilate_kernel_size1: 32548
gua_kernal_size: 50
catch_num: 48267
hsv_h_max: 180
button: 1
rgb_b_max: 255
threhold1: 100
rgb_g_max: 255
hsv_s_min: 255
rgb_r_min: 255
erode_kernel_size1: 3
row: 383
hsv_v_max: 255
threhold2: 100
col: 5
rgb_b_min: 255
hsv_v_min: 0
hsv_s_max: 255
rgb_g_min: 255
rgb_r_max: 255
hsv_h_min: 180